examples/basic/android/app/build.gradle

```diff
@@ -81,12 +81,20 @@ android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
     namespace "com.videoplayer"
+
+    compileOptions {
+        // These options are necessary to be able to build fro source
+        coreLibraryDesugaringEnabled true
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
+    }
```

Comment:
Shouldn't this be mentioned in docs ?

Reply:
Yes you are right, but this is already in this section:
### Desugaring
I pointed the google documentation instead of copying it !